server/plugin/plugin.go

```diff
@@ -84,7 +84,22 @@ func NewPlugin() *Plugin {
 	return p
 }
 
-func (p *Plugin) githubConnect(token oauth2.Token) *github.Client {
+func (p *Plugin) githubConnectUser(ctx context.Context, info *GitHubUserInfo) *github.Client {
+	access := info.Token.AccessToken
+	config := p.getConfiguration()
+	updated, err := p.forceResetUserTokenMM34646(ctx, config, *info)
```

Comment:
I think the performance impact of this is fine. On a update of the server all connected clients will reset there tokens meaning `#Users` requests are made. But they will be made anyway, as clients tend to refresh after an update.

Reply:
2/5 Good point (active clients resetting immediately) - We may want to delay the global reset script by a few minutes to get the active users a chance to reset first?

Reply:
Good idea :+1: